internal/compiler/parser/expressions.rs

```diff
@@ -427,6 +430,102 @@ fn parse_tr(p: &mut impl Parser) {
     p.expect(SyntaxKind::RParent);
 }
 
+#[cfg_attr(test, parser_test)]
+/// ```test,AtKeys
+/// @keys()
+/// @keys(x)
+/// @keys(ctrl +shift + alt+meta+a)
+/// @keys(control +shift + alt+meta+a)
+/// ```
+fn parse_keys(p: &mut impl Parser) {
```

Comment:
This should allow `,` separated lists of elements.

The logic to decide on the actual key being uses is also a bit simplistic: We should (in addition?) accept the keys from our `Keys` namespace. But So I should probably let all unknown identifiers through here and match them to the Keys namespace (or the single character).